src/helpers/ScrollToTop/ScrollToTop.jsx

```diff
@@ -1,4 +1,4 @@
-import React, { Component } from 'react';
+import { Component } from 'react';
```

Comment:
I don't know if you can do this in React...
Better:
```
import React from 'react';

...

class bla extends React.Component {...
```

Reply:
We need to define which way we prefer. I don't care either way, we just have to make a decision. If importing React is the way to go, I am fine with it. In this case the "React" import was not used at all.

Reply:
Fixed. We still kind of need to define that somewhere, or better find a linter that checks this.